src/screens/rewards/components/RewardsStatsCard.tsx

```diff
@@ -25,6 +25,32 @@ export const RewardsStatsCard: React.FC<Props> = ({
 }) => {
   const infoIconColor = useInfoIconColor();
 
+  if (!onPress) {
```

Comment:
i think this would look cleaner with the `ConditionalWrap` component. you can also just keep the `ButtonPressAnimation` but add the prop `disabled={!onPress}`